subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BaseDeprecations.groovy

```diff
@@ -55,7 +55,7 @@ class BaseDeprecations {
     public static final String GUTIL_DEPRECATION = "The org.gradle.util.GUtil type has been deprecated. " +
         "This is scheduled to be removed in Gradle 9.0. " +
         "Consult the upgrading guide for further information: " +
-        DOCUMENTATION_REGISTRY.getDocumentationFor("upgrading_version_8","base_convention_deprecation")
+        "${DOCUMENTATION_REGISTRY.getDocumentationFor("upgrading_version_7", "org_gradle_util_reports_deprecations")}"
```

Comment:
❔ Why "${}" instead of plain call?

Reply:
Good catch, I copy pasted the message when extracting and IDEA did this 🙄 
Fixed!